src/dlgConnectionProfiles.cpp

```diff
@@ -1670,6 +1682,26 @@ void dlgConnectionProfiles::fillout_form()
         }
     }
 
+
+    mudServer = QStringLiteral("Fierymud");
+    if (!deletedDefaultMuds.contains(mudServer)) {
+        pM = new QListWidgetItem(mudServer);
+        pM->setFont(font);
+        pM->setForeground(QColor(Qt::white));
+        profiles_tree_widget->addItem(pM);
+        if (!hasCustomIcon(mudServer)) {
+            mi = QIcon(QPixmap(QStringLiteral(":/icons/fiery_mud.png")));
```

Comment:
No! You do NOT need to go via a `QPixmap` intermediary, you can use the file name directly as the icon as:
```cpp
             mi = QIcon(QStringLiteral(":/icons/fiery_mud.png"));
```